README.md

```diff
@@ -22,3 +22,155 @@ e.g.
 ```
 npm i @fastify/cors -w @mykomap/back-end --save
 ```
+
+## Installation and updates
```

Comment:
There are a lot of details here so I think they should be moved to a `docs/deployment.md` file to avoid the README getting clogged up. Then we can include a link in the README to this doc.